src/oss/schemas/Replication.yml

```diff
@@ -0,0 +1,32 @@
+type: object
+properties:
+  id:
+    type: string
+  name:
+    type: string
+  description:
+    type: string
+  remoteID:
+    type: string
+  localBucketID:
+    type: string
+  remoteBucketID:
+    type: string
+  maxQueueSizeBytes:
+    type: int
+    format: int64
+  currentQueueSizeBytes:
+    type: int
+    format: int64
+  latestResponseCode:
+    type: int
+  latestErrorMessage:
+    type: string
+required:
```

Comment:
Does this need a property for "skip-verify/insecure TLS"?

Reply:
Nope, that's captured in the `RemoteConnection` referenced by `remoteID`